src/models/databaseModel.ts

```diff
@@ -297,272 +37,26 @@ export enum DbStatePathType {
-    dbState.type === DbStateType.XMR_LOCK_PROOF_RECEIVED
-  );
-}
-
-export interface MergedXmrLockedDbState extends MergedDbState {
-  type: DbStateType.XMR_LOCKED;
-  state: ExecutionSetupDoneDbState &
-    BtcLockedDbState &
-    XmrLockProofReceivedDbState &
-    XmrLockedDbState;
-}
-
-export function isMergedXmrLockedDbState(
-  dbState: MergedDbState
-): dbState is MergedXmrLockedDbState {
-  return (
-    isExecutionSetupDoneDbState(dbState.state) &&
-    isBtcLockedDbState(dbState.state) &&
-    isXmrLockProofReceivedDbState(dbState.state) &&
-    isXmrLockedDbState(dbState.state) &&
-    dbState.type === DbStateType.XMR_LOCKED
-  );
-}
-
-export interface MergedEncSigSentDbState extends MergedDbState {
-  type: DbStateType.ENC_SIG_SENT;
-  state: ExecutionSetupDoneDbState &
-    BtcLockedDbState &
-    XmrLockProofReceivedDbState &
-    XmrLockedDbState &
-    EncSigSentDbState;
-}
-
-export function isMergedEncSigSentDbState(
-  dbState: MergedDbState
-): dbState is MergedEncSigSentDbState {
-  return (
-    isExecutionSetupDoneDbState(dbState.state) &&
-    isBtcLockedDbState(dbState.state) &&
-    isXmrLockProofReceivedDbState(dbState.state) &&
-    isXmrLockedDbState(dbState.state) &&
-    isEncSigSentDbState(dbState.state) &&
-    dbState.type === DbStateType.ENC_SIG_SENT
-  );
-}
-
-export interface MergedBtcRedeemedDbState extends MergedDbState {
-  type: DbStateType.BTC_REDEEMED;
-  state: ExecutionSetupDoneDbState &
-    BtcLockedDbState &
-    XmrLockProofReceivedDbState &
-    XmrLockedDbState &
-    EncSigSentDbState &
-    BtcRedeemedDbState;
-}
-
-export function isMergedBtcRedeemedDbState(
-  dbState: MergedDbState
-): dbState is MergedBtcRedeemedDbState {
-  return (
-    isExecutionSetupDoneDbState(dbState.state) &&
-    isBtcLockedDbState(dbState.state) &&
-    isXmrLockProofReceivedDbState(dbState.state) &&
-    isXmrLockedDbState(dbState.state) &&
-    isEncSigSentDbState(dbState.state) &&
-    isBtcRedeemedDbState(dbState.state) &&
-    dbState.type === DbStateType.BTC_REDEEMED
-  );
-}
-
-export interface MergedDoneXmrRedeemedDbState extends MergedDbState {
-  type: DbStateType.DONE_XMR_REDEEMED;
-  state: ExecutionSetupDoneDbState &
-    BtcLockedDbState &
-    XmrLockProofReceivedDbState &
-    XmrLockedDbState &
-    EncSigSentDbState &
-    BtcRedeemedDbState &
-    DoneXmrRedeemedDbState;
-}
-
-export function isMergedDoneXmrRedeemedDbState(
-  dbState: MergedDbState
-): dbState is MergedDoneXmrRedeemedDbState {
-  return (
-    isExecutionSetupDoneDbState(dbState.state) &&
-    isBtcLockedDbState(dbState.state) &&
-    isXmrLockProofReceivedDbState(dbState.state) &&
-    isXmrLockedDbState(dbState.state) &&
-    isEncSigSentDbState(dbState.state) &&
-    isBtcRedeemedDbState(dbState.state) &&
-    isDoneXmrRedeemedDbState(dbState.state) &&
-    dbState.type === DbStateType.DONE_XMR_REDEEMED
-  );
+export function getSwapTxFees(swap: GetSwapInfoResponse): number {
+  // TODO
+  return 124;
 }
```

Comment:
**Suggestion:** The function `getSwapTxFees` has a TODO comment and returns a hardcoded value. It would be better to implement the function or, if it's not possible at the moment, throw an error or return a more meaningful default value.
```suggestion
export function getSwapTxFees(swap: GetSwapInfoResponse): number {
  throw new Error('Function getSwapTxFees not implemented yet');
}
```